mmv1/templates/tgc_next/test/test_file.go.tmpl

```diff
@@ -0,0 +1,36 @@
+// ----------------------------------------------------------------------------
+//
+//     ***     AUTO GENERATED CODE    ***    Type: MMv1     ***
+//
+// ----------------------------------------------------------------------------
+//
+//     This file is automatically generated by Magic Modules and manual
+//     changes will be clobbered when the file is regenerated.
+//
+//     Please read more about how to change this file in
+//     .github/CONTRIBUTING.md.
+//
+// ----------------------------------------------------------------------------
+
+package {{$.PackageName}}_test
+
+import (
+	"testing"
+
+	"github.com/GoogleCloudPlatform/terraform-google-conversion/v6/test"
+)
+
+{{ range $e := $.TestExamples }}
+func TestAcc{{ $e.TestSlug $.ProductMetadata.Name $.Name }}(t *testing.T) {
+	t.Parallel()
+
+	test.BidirectionalConversion(
+		t,
+		[]string{
+{{- range $field := $e.IgnoreTestPropertiesTGC }}
```

Comment:
`$e.IgnoreTestPropertiesTGC`  should be replaced with the new function mentioned in the above comment.

Reply:
```suggestion
{{- range $field := $.IgnoreTestPropertiesTGC }}
```